### PR TITLE
Add example for filtering across associations

### PR DESCRIPTION
**Pull request by chevinbrown**
This is a _very_ basic example of how to filter across associations. It works for me, but there may be a more clear way to do this, especially with Ecto 3 hot off the press. 😄 

@rcdilorenzo sorry this promise took so long!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,8 +92,61 @@ With this example, here are some of the query parameters that can be used:
...
+    # step 2: build query
+    user_query =
+      User
+      |> Filtrex.query(user_filter)
+
+    profile_query =
+      Profile
+      |> Filtrex.query(profile_filter)
+
+    query =
```
Comment (rcdilorenzo) [inline README.md:118]: Let's add a separation comment here to remain consistent.
```suggestion
    # step 3: combine queries
    query =
```

File: README.md [hunk 2]
```diff
@@ -92,8 +92,61 @@ With this example, here are some of the query parameters that can be used:
 | `posted_at_between` (nested value)  | start: "01-01-2013" <br> end: "12-31-2017" |
 | `filter_union` (any \| all \| none) | any                                        |
 
+## Filtering across associations
```
Comment (rcdilorenzo) [inline README.md]: Adjust capitalization for consistency
```suggestion
## Filtering Across Associations
```

**Review by rcdilorenzo — CHANGES_REQUESTED** [review]
Thanks so much @chevinbrown. I've suggested a couple of super minor changes, but otherwise looks good. If you make the changes, I'll try and merge it tonight in case you're pursuing Octoberfest. 😄

**Comment by chevinbrown** [comment]
@rcdilorenzo looks good to me! 💸

**Pull request merged**